sdk/armada-protocol-common/src/deployments/index.ts

```diff
@@ -114,3 +114,36 @@ export const getDeployedGovRewardsManagerAddress = () => {
   }
   return Address.createFromEthereum({ value: maybeAddress })
 }
+
+export const getBridgeContractAddress = (chainInfo: ChainInfo) => {
+  const config = getConfig()
+  const key = getConfigKey(chainInfo.name)
+  return Address.createFromEthereum({
+    value: config[key].deployedContracts.gov.summerToken.address,
+  })
+}
```

Comment:
This seem like a redundancy, you can get summer token addresss in Armada manager using: 
      `ArmadaManager.getSummerToken` or this:
      ```
          const address = getDeployedContractAddress({
      chainInfo: params.chainInfo,
      contractCategory: 'gov',
      contractName: 'summerToken',
    })
    
```